rust/noosphere-cli/tests/peer_to_peer.rs

```diff
@@ -14,6 +14,7 @@ use noosphere_sphere::{
     SpherePetnameRead, SpherePetnameWrite, SphereReplicaRead, SphereSync, SphereWalker,
     SyncRecovery,
 };
+use rand::Rng;
```

Comment:
Any concerns w/ non-determinism in the test suite as a result of this?

Reply:
Yes, although it's intentional in this case. The test writes to random keys (within a range) in the sphere. My intention is to simulate different probabilistic distributions.

We can enhance this by using a seed, and report the seed when running the test. At least that way we can reproduce the failure if we get a flake. I'll add that before we land this.

Reply:
The RNG is now seeded, and the seed in use is reported in the test logs